src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -33,71 +42,106 @@
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  * @author Saša Stamenković <umpirsky@gmail.com>
  */
-class ResourceController extends FOSRestController
+class ResourceController extends ContainerAware
```

Comment:
I'd rather discourage using them. Especially when we start to properly (not yet) inject deps into Controller. I know that we still leave ContainerAware and inject whole container to keep things simple for Symfony folks. I need to think about how much of a compromise we can do here. :) Maybe it makes sense to just extend Controller.